security/src/main/java/nl/nn/adapterframework/lifecycle/servlets/ServletAuthenticatorBase.java

```diff
@@ -159,7 +155,10 @@ private SecurityFilterChain createSecurityFilterChain() {
 	public SecurityFilterChain configureHttpSecurity(HttpSecurity http) {
 		try {
 			//Apply defaults to disable bloated filters, see DefaultSecurityFilterChain.getFilters for the actual list.
-			http.headers().frameOptions().sameOrigin(); //Allow same origin iframe request
+			http
+			.headers()
+			.frameOptions()
+			.sameOrigin(); //Allow same origin iframe request
```

Comment:
Niels would mostly not like these 4 lines being changes ;-)

Reply:
Oh boi 😢